pkg/network/go/bininspect/dwarf.go

```diff
@@ -132,7 +131,11 @@ func (d dwarfInspector) findFunctionDebugInfoEntries(functions []string) (map[st
 	}
 
 	if len(functionsToSearch) != 0 {
-		return nil, errors.New("not all functions found")
+		missingFunctions := make([]string, 0, len(functionsToSearch))
+		for funcName := range functionsToSearch {
+			missingFunctions = append(missingFunctions, funcName)
+		}
+		return nil, fmt.Errorf("not all functions found: %v", missingFunctions)
 	}
```

Comment:
didnt bryce do the exact same change in his PR?

Reply:
I just went through his PRs (he opened 5 recently) and didn't see it, can you please link what you're referring to.

Reply:
https://github.com/DataDog/datadog-agent/pull/36771#discussion_r2078797192

Reply:
It makes more sense in this PR, rather than mine which didn't touch DWARF otherwise.